.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 
 node_js:
+  - iojs
+  - '0.12'
```

Comment:
@tinganho stupid question, does it run the tree targets all the time? i.e. it would not considered passed until the test passed on the three targets? if so i would keep it to two if we can.